Docs on how non teams can contribute to the project. (#850)
sFractal-Podii/quizquadaminos: docs/contributing.md

This project uses [semantic versioning](https://semver.org/) as much as we can

## Contributing for Non team.
### Open an issue

Visit github issues (https://github.com/sFractal-Podii/quizquadaminos/issues)  and ensure that the issues you want to create does not exist before creating the issue.

### Open a pull request

> To create a pull request :

Reviewer: Rectify the numbering

1. Fork the project and clone it locally.https://github.com/sFractal-Podii/quizquadaminos.
1. Create your branch for your changes. Naming of the branch should be according to the issue you are working on.
1.  Make the changes and create the pull request. Compare your branch on the fork against the master branch of the original repo.

> ### After creating a Pull request.

Reviewer: You can add:
- Issues are merged once they are approved
- For clean history of commits, they should rebase if they have more than one commit

This project uses [semantic versioning](https://semver.org/) as much as we can

## Contributing for Non team.
### Open an issue

Visit github issues (https://github.com/sFractal-Podii/quizquadaminos/issues)  and ensure that the issues you want to create does not exist before creating the issue.

### Open a pull request
To create a pull request :
1. Fork the project and clone it locally.https://github.com/sFractal-Podii/quizquadaminos.
2. Create your branch for your changes. Naming of the branch should be according to the issue you are working on.

> 3.  Make the changes and create the pull request. Compare your branch on the fork against the master branch of the original repo.

Reviewer: ```suggestion
3.  Make the changes and create the pull request. Compare your branch on the fork against the develop branch of the original repo.
```

### After creating a Pull request.
- Constantly visit your pull request to see the comments left and work on the comments.
- Issues are merged once they are approved

> - For clean history of commits, they should rebase if they have more than one commit

Reviewer: ```suggestion
- For clean history of commits, rebase if the PR have more than one commit
```
